index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <!-- <h1>Hello World! We are software developers</h1>  -->
+  <nav id="nav-bar" class="header">
+    <ui>
+      <li><a href="About-Us">About Us</a></li>
+      <li><a href="Career">Career</a></li>
+      <li><a href="Location">Location</a></li>
```

Comment:
## Hover effects
Kindly apply hover effects to the link as stated in the instructions.